README.md

```diff
@@ -1,284 +1,305 @@
+- [Test Suite](#test-suite)
+  - [Unit tests](#unit-tests)
+  - [Functional tests](#functional-tests)
+- [Contributing](#contributing)
+- [License](#license)
 
-## What's inside
+## Overview
+`pip-account-management` is a microservice that deals with operations related to accounts, including interaction with Azure B2C for PI_AAD users. It also manages the audit functionality.
 
-The template is a working application with a minimal setup. It contains:
- * application skeleton.
- * setup script to prepare project
- * common plugins and libraries.
- * docker setup.
- * swagger configuration for api documentation ([see how to publish your api documentation to shared repository](https://github.com/hmcts/reform-api-docs#publish-swagger-docs))
- * code quality tools already set up.
- * integration with Travis CI.
- * Hystrix circuit breaker enabled
- * MIT license and contribution information.
- * Helm chart using chart-java.
+It sits within the Court and Tribunal Hearings Service (CaTH hereafter), written with Spring Boot/Java.
 
-The application exposes health endpoint (http://localhost:6969/health) and metrics endpoint
-(http://localhost:6969/metrics).
+In practice, the service is usually containerized within a hosted kubernetes environment within Azure.
 
-## Plugins
+All interactions with `pip-account-management` are performed through the API (specified in [API Documentation](#api-documentation)) either as a standalone service or via connections to other microservices.
 
-The template contains the following plugins:
+## Features and Functionality
 
-  * checkstyle
+- Creation of new verified PI_AAD users in Azure B2C and in the PiUser postgres table.
+- Creation of new system admin accounts in Azure B2C and in the PiUser postgres table, and ensures the limit is not breached.
+- For third parties and other IDAMs (e.g. CFT IDAM), creation of the user in the PiUser postgres database.
+- Maintenance of the above users, such as updating roles and deletion of account and subscriptions when expired.
+- Provides functionality to check if a user has permission to see a list based on the sensitivity and list type.
+- Allows the creation of an audit log to log admin activities.
+- Provides the ability to bulk create verified users (follows the same process as point 1)
+- Triggers the sending of emails to users under various conditions (e.g New user, expiring, audit actions).
+- Manages the account application process for PI_AAD users.
+- Provides MI reporting endpoints, which are used to produce the MI report.
+- Flyway for database modifications via SQL ingestion.
+- Secure/Insecure Mode: Use of bearer tokens for authentication with the secure instance (if desired)
+- OpenAPI Spec/Swagger-UI: Documents and allows users or developers to access API resources within the browser.
+- Integration tests using TestContainers for dummy database operations.
 
-    https://docs.gradle.org/current/userguide/checkstyle_plugin.html
+## Azure B2C integration
```

Comment:
Most of the main headings have uppercase first letter. So may be if should be "**I**ntegration"?